module/People/src/People/View/UserProfileJsonModel.php

```diff
@@ -36,7 +36,7 @@ public function serialize()
 									'name'=>$organization->getName(),
 								],
 								'role'=>$role,
-								'createdAt'=>$membership->getCreatedAt(),
+								'createdAt'=>$membership->getCreatedAt()->format("d/m/Y"),
```

Comment:
Il formato di data di una struttura JSON non è quello italiano! In uno qualsiasi degli altri *JsonModel puoi vedere quale deve essere